extensions/amp-consent/0.1/consent-config.js

```diff
@@ -23,8 +23,7 @@ import {deepMerge, hasOwn, map} from '#core/types/object';
 import {devAssert, user, userAssert} from '../../../src/log';
 import {getChildJsonConfig} from '#core/dom';
 
-const TAG = 'amp-consent/consent-config';
-const AMP_STORY_CONSENT_TAG = 'amp-story-consent';
+const TAG = 'amp-consent';
```

Comment:
Somewhat unrelated, but I'm removing this "amp-consent/whatever" `TAG` format generally since it's exposed in user warnings.